Makefile

```diff
@@ -57,6 +57,14 @@ BABEL := $(NODE_LOCAL_BIN)/babel
 ESLINT := $(NODE_LOCAL_BIN)/eslint
 FLAKE8 := $(NODE_LOCAL_BIN)/flake8
 
+ifdef DEV_XPI
```

Comment:
Maybe add a comment about how this is expected to be set?  (eg the command-line)